ios/Classes/BetterPlayer.m

```diff
@@ -71,6 +72,11 @@ - (void)addObservers:(AVPlayerItem*)item {
                                                  selector:@selector(itemFailedToPlayToEndTime:)
                                                      name:AVPlayerItemFailedToPlayToEndTimeNotification
                                                    object:item];
+        
+        // use CMTimeMake(1000000, 1) to only invoking the block whenever time jumps or playback starts or stops. (Doc: https://developer.apple.com/documentation/avfoundation/avplayer/1385829-addperiodictimeobserverforinterv#discussion)
```

Comment:
About `CMTimeMake(1000000, 1)`,
You want,
- Don't call `usingBlock` 's function as the interval.
- Call `usingBlock` 's function when "time jumps or playback starts or stops".
right?

We cannot use `CMTime.positiveInfinity`?
https://developer.apple.com/documentation/coremedia/cmtime/1400789-positiveinfinity

Reply:
> About CMTimeMake(1000000, 1),
> You want,
> 
> Don't call usingBlock 's function as the interval.
> Call usingBlock 's function when "time jumps or playback starts or stops".
> right?

Yes, you got it right.

> We cannot use CMTime.positiveInfinity?

Yes, we can't. We'll got an exception:

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[AVPeriodicTimebaseObserver initWithTimebase:interval:queue:block:] invalid parameter not satisfying: ((Boolean)(((interval).flags & (kCMTimeFlags_Valid | kCMTimeFlags_ImpliedValueFlagsMask)) == kCMTimeFlags_Valid))'
```

Reply:
Oh...
Thank you for checking. I understand. 👍